File: uiTestExpresso/app/src/main/java/com/example/uitestexpresso/MainActivity.java
package com.example.uitestexpresso;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private Button btnChange;
    private EditText etTextToChange;
    private TextView tvChangedText;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnChange =  findViewById(R.id.btnChange);

        etTextToChange =  findViewById(R.id.etTextToChange);

        tvChangedText =  findViewById(R.id.tvChangedText);

        btnChange.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                tvChangedText.setText(etTextToChange.getText());
            }
        });

    }
}

File: KotlinUiTest/app/src/main/java/com/example/kotlinuitest/MainActivity.kt
package com.example.kotlinuitest

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val button:Button= findViewById(R.id.buttonval);

        button.setOnClickListener(View.OnClickListener {
            Toast.makeText(this,"Testing is going",Toast.LENGTH_SHORT).show()

        })

    }
}